node-installer/integration-test-minikube.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "=== Step 1: Create a MiniKube cluster ==="
+minikube start -p spin-minikube --driver=docker --container-runtime=containerd
```

Comment:
There is this github action for setting up minikube if we are running into issues here https://minikube.sigs.k8s.io/docs/tutorials/setup_minikube_in_github_actions/